README.md

```diff
@@ -52,7 +52,7 @@ open Freetime.xcworkspace/
 
 This will install [apollo-codegen](https://github.com/apollographql/apollo-codegen) that is required to generate the GraphQL models.
 
-See the [setup guide](Setup.md) for details about API keys and environment variables.
+See the [setup guide](https://github.com/GitHawkApp/GitHawk/blob/master/Setup.md) for details about API keys and environment variables.
```

Comment:
Why did you change this one?

Reply:
Didn’t work in GitHawk. Opened to GitHub 404 error

Reply:
Is there a bug raised for this?